requirements.txt

```diff
@@ -0,0 +1,4 @@
+numpy
+sympy
+torch
```

Comment:
does this work now? no need to pick a specific package?

Reply:
yep that works. See pytorch.org